templates/partials/committee-charts.html

```diff
@@ -1,3 +1,6 @@
+{% with %}
```

Comment:
This `with` block breaks the page--`reports` and `totals` are added directly to `tmpl_vars` in `views.py`, lines 98-99. They aren't attached to a `financials` dict, so this lookup fails. The page seems to work fine if we delete the `with`, `set`, and `endwith` statements.